modules/monitoring-creating-cluster-id-labels-for-metrics.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="creating-cluster-id-labels-for-metrics_{context}"]
+= Creating cluster ID labels for metrics
+
+You can configure cluster ID labels for metrics for default platform monitoring and for user workload monitoring.
```

Comment:
For minimalism and user focus, can you omit, `You can` and simply say, `Configure cluster ID labels for metrics for default platform and user workload monitoring.` 

Note, if default platform and user workload have _separate_ monitoring, then keep it how you have written. If share monitoring, no need to say it twice, you can use `monitoring` once like I did.

Reply:
They are separate monitoring systems, so leaving as-is.